gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatability.kt

```diff
@@ -0,0 +1,14 @@
+package org.jetbrains.compose
+
+internal object ComposeCompilerCompatability {
+    fun compilerVersionFor(kotlinVersion: String): ComposeCompilerVersion? = when (kotlinVersion) {
+        "1.7.10" -> ComposeCompilerVersion("1.3.0-alpha01")
+        "1.7.20" -> ComposeCompilerVersion("1.3.2-alpha01", isJsSupported = false)
```

Comment:
1.3.0-alpha01 and 1.3.2-alpha01
Maybe we can you stable versions?
https://mvnrepository.com/artifact/androidx.compose.compiler/compiler

Reply:
Yes, we will release stable versions and use them here, when we will be sure that everything works.